modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -111,9 +111,9 @@ zypper in spacewalk-utils
 Adjust the version of the channel names to match your {ubuntu} version:
 +
 ----
-spacewalk-common-channels ubuntu-1804-amd64-main \
-ubuntu-1804-amd64-security \
-ubuntu-1804-amd64-updates
+spacewalk-common-channels -u admin -p pass -a amd64-deb ubuntu-1804-amd64-main \
+ubuntu-1804-amd64-main-updates \
```

Comment:
Is it `updates` or `update`? If the first, then I am pretty sure you need to adjust the equivalent name for Uyuni below, as it is called `ubuntu-1804-amd64-main-update-uyuni `